packages/acs-ui-common/package.json

```diff
@@ -34,16 +34,16 @@
     "@azure/logger": "1.0.3"
   },
   "peerDependencies": {
-    "react": ">=16.8.0 <18.0.0",
-    "@types/react": ">=16.8.0 <18.0.0"
+    "react": ">=16.8.0 <19.0.0",
+    "@types/react": ">=16.8.0 <19.0.0"
   },
   "devDependencies": {
     "@babel/cli": "~7.16.0",
     "@babel/core": "~7.16.0",
     "@microsoft/api-documenter": "~7.12.11",
     "@microsoft/api-extractor": "~7.18.0",
     "@types/jest": "~26.0.22",
-    "@types/react": "^16.9.49",
+    "@types/react": "18.2.0",
```

Comment:
Do we need to run react 16 compatibility test by any chance?
BTW, why not use ^ to get latest update for react 18

Reply:
This should be pinned to the same as react so this should be whatever react is. Really they should probably all be `~` or `^` but we don't have good repo-wide guidenence on what developers should choose. Should we set a standard now and update guidence in the wiki?

Reply:
And yeah we will need that, I have something cooking up on the side but haven't had time for it, will be similar to the typescript check
![image](https://github.com/Azure/communication-ui-library/assets/2684369/13b12576-07c7-4774-8c5d-01bc8d547d6c)

Reply:
> 

Yea, would be nice to always get latest bit of the current compatible version, cause it might be something unexpected happening there (very rare though) then we can catch it earlier than our customer (cause customers prefer those "^" for versions)

Reply:
I will keep this change for a separate PR, made a bug for it for tracking [Bug 3279379](https://skype.visualstudio.com/SPOOL/_workitems/edit/3279379): [Web] Repo should have guidence on using a pinned dep version vs `^` or `~`